### DISCUSSION TITLE
TypedStorage is deprecated warning causing the generations to become extremely slow

### DISCUSSION THREAD

**Discussion by Aurki01** [lllyasviel/Fooocus#3465] [Q&A] [answered]
hey, since yesterday i started receiving a warning message on startup of fooocus:
`D:\Fooocus\python_embeded\lib\site-packages\torch\_utils.py:831: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()`
and it's taking me 5 to 6 times more time than before for each generation, i'm not sure if this is the cause for it and if it is how may i solve this
would really appreciate some help!!

**Comment by mashb1t** [comment, marked as answer]
This is just a deprecation hint and has nothing to do with slow downs or any application behavior. Feel free to open an issue if you have any and provide further details.

**Reply by Aurki01** [reply]
i see, thanks for clarifying 
i was confused since it used to take me 4-5 mins per generation before but since yesterday I started seeing that warning and now it's taking me 20 mins for each generation 
I'll try and look what could be the other cause

**Reply by mikasaq** [reply]
Hello @Aurki01,  have you managed to solve this issue?
I'm encountering a similar warning:
`/home/user/.conda/envs/env_name/lib/python3.10/site-packages/torch/_utils.py:831: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()  return self.fget.__get__(instance, owner)(), `
It seems to make the process extremely slow on my end.
Do you have any suggestions or workaround for this?

**Reply by Aurki01** [reply]
hey @mikasaq as mash mentioned it was just a msg nothing else, the problem i was facing was that I'm using a weak 6gb vram gpu and i was not providing enough space for paging file which helps in increasing generation speed, now i have allocated around 50gb space from my ssd and I'm able to generate an image in around 3 minutes. If u have a weak hardware like mine try providing more paging space through ur ssd and that might just work for u

**Reply by mikasaq** [reply]
Thank you for your detailed and patient response.
